appserver/payara-appserver-modules/jmx-monitoring-service/src/main/java/fish/payara/jmx/monitoring/admin/GetJMXMonitoringConfiguration.java

```diff
@@ -167,12 +166,9 @@ public void execute(AdminCommandContext context) {
         actionReport.setExtraProperties(extraProps);
 
         if (!monitoringConfig.getNotifierList().isEmpty()) {
-            List<Class<Notifier>> notifierClassList = Lists.transform(monitoringConfig.getNotifierList(), new Function<Notifier, Class<Notifier>>() {
-                @Override
-                public Class<Notifier> apply(Notifier input) {
-                    return resolveNotifierClass(input);
-                }
-            });
+            List<Class<Notifier>> notifierClassList = monitoringConfig.getNotifierList().stream().map((input) -> {
+                return resolveNotifierClass(input);
```

Comment:
could just be `input -> resolveNotifierClass(input);` without the curly and round braces or even `this::resolveNotifierClass` (or even `GetJMXMonitoringConfiguration::resolveNotifierClass`).